english/watch-movie/Up.md

```diff
@@ -3,4 +3,6 @@ Halfway through the movie, the marshmallow walks out.
 You change the TV to F-Troop and it comes back in and sits down.  
 Turns out it loves Agarn!
 You then change the channel to meet the press
-Marshmallow gets even more fascinated
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.